src/pickup.cpp

```diff
@@ -89,10 +89,18 @@ interact_results interact_with_vehicle( vehicle *veh, const tripoint &pos,
     } options;
     uimenu selectmenu;
 
-    selectmenu.addentry( EXAMINE, true, 'e', _( "Examine vehicle" ) );
+    if( !veh->is_furniture ) {
```

Comment:
Seems the only difference here is the string itself. Why duplicate all the surrounding stuff?

You can use the ternary operator for this:
```C++
addentry( ..., veh->is_furniture ? _( "Examine furniture" ) : _( "Examine vehicle" ) );
```